waveprop/rs.py

```diff
@@ -916,7 +936,7 @@ def _bandpass(H, fX, fY, Sx, Sy, x0, y0, z0, wv):
     :param y0:
     :return:
     """
-    du = 1 / (Sx) # not divided by 2 like in 17 since field already padded
+    du = 1 / (2 * Sx)
```

Comment:
are you sure about multiplying by 2? Sx and Sy are already padded when given as an input: https://github.com/ebezzam/waveprop/blob/84c3e7dd50c7193e4c069a7581740ecbf33a3167/waveprop/rs.py#L864

I agree that naming could have been better. Dx and Dy might have been better (D for destination plane)